src/actions/notification.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import { NOTIFICATION_SUCCESS } from '../actionTypes/notification'
+
+
+const API = process.env.REACT_APP_API;
+const { token } = localStorage;
+
+export const success = (notifications) => ({
+    type: NOTIFICATION_SUCCESS,
+    payload: notifications,
+})
+
+export const fetchNotifications = () => (dispatch) => axios.get(`${API}/api/users/notifications`, {
+    headers: {
+        Authorization: `Bearer ${token}`,
+        "Content-Type": "application/json"
+    }
+}).then((response) => {
+    return dispatch(success(response.data));
+}).catch();
```

Comment:
You should dispatch an error here. The reasons why notification did not come should be displayed on the notification dropdown so the user is more aware of what's going on.